src/Symfony/Component/HttpClient/Response/CurlResponse.php

```diff
@@ -230,8 +234,10 @@ protected static function schedule(self $response, array &$runningResponses): vo
 
     /**
      * {@inheritdoc}
+     *
+     * @param CurlClientState $multi
```

Comment:
you can use a real type here instead, isn't it?

Reply:
In fact, I can. Will change it.